credentials/apps/edx_credentials_extensions/edly_credentials_app/views.py

```diff
@@ -0,0 +1,3 @@
+"""
```

Comment:
similarly we don't need this file unless there actually is a view

Reply:
similarly, here kept it there if in future we need to use this.